src/main/steps/applicant1/hub-page/post.ts

```diff
@@ -1,11 +1,24 @@
 import autobind from 'autobind-decorator';
 
-import { APPLICANT_1_CONFIRM_RECEIPT } from '../../../app/case/definition';
+import {
+  APPLICANT_1_CONFIRM_RECEIPT,
+  APPLICANT_2_CONFIRM_RECEIPT,
+  DRAFT_CONDITIONAL_ORDER,
+  State,
+  UPDATE_CONDITIONAL_ORDER,
+} from '../../../app/case/definition';
+import { AppRequest } from '../../../app/controller/AppRequest';
 import { AnyObject, PostController } from '../../../app/controller/PostController';
 
 @autobind
 export default class HubPagePostController extends PostController<AnyObject> {
-  protected getEventName(): string {
-    return APPLICANT_1_CONFIRM_RECEIPT;
+  protected getEventName(req: AppRequest): string {
+    if (req.session.userCase.state === State.Holding) {
+      return req.session.isApplicant2 ? APPLICANT_2_CONFIRM_RECEIPT : APPLICANT_1_CONFIRM_RECEIPT;
+    } else if (req.session.userCase.state === State.AwaitingConditionalOrder) {
+      return DRAFT_CONDITIONAL_ORDER;
+    } else {
+      return UPDATE_CONDITIONAL_ORDER;
```

Comment:
would this ever be called from the hub page?

Reply:
if app1 has already started CO they would have triggered DRAFT_CONDITIONAL_ORDER, so the case state would be ConditionalOrderDrafted, when app2 then logs in and starts CO from the hub page they should trigger UPDATE_CONDITIONAL_ORDER as the CO has already been drafted